packages/storybook-vue/stories/components/beta-brand-header-navigation/SlotLangSwitcher.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <scale-telekom-nav-list variant="lang-switcher" slot="lang-switcher" alignment="right">
-    <scale-telekom-nav-item active>
-      <a href="#">EN</a>
+  <scale-telekom-nav-list variant="lang-switcher" slot="lang-switcher" alignment="right" aria-label="Language switcher">
+    <scale-telekom-nav-item active aria-label="Switch to English">
+      <a href="#" title="English">EN</a>
     </scale-telekom-nav-item>
-    <scale-telekom-nav-item>
-      <a href="#">SO</a>
+    <scale-telekom-nav-item aria-label="Switch to Somali">
```

Comment:
I'm maybe too picky here but I would change Somali for some other language where this might be actually present 😁, maybe `hu` for Hungarian?

Reply:
yea i was just following the designs, not sure why SO is used there XD (I also se EN changed to MK, Macedonian, for some reason). But it's fine for me to update it